resources/views/emails/membership/approved.blade.php

```diff
@@ -6,6 +6,11 @@
 
 The next step in setting up your hackspace membership is for you to set up a bank standing order to Nottingham Hackspace using the details below.
 
+As explained on your tour around the Hackspace, we have a "Pay what you think the space is worth to you" system. We are entirely member funded and rely almost exclusively on membership contributuons to keep the Hackspace open. 
```

Comment:
You will want a \<br\> tag at the end of this line to keep the line break/paragraph style when in html